packages/react-devtools-scheduling-profiler/babel.config.js

```diff
@@ -0,0 +1,44 @@
+const chromeManifest = require('../react-devtools-extensions/chrome/manifest.json');
+const firefoxManifest = require('../react-devtools-extensions/firefox/manifest.json');
+
+const minChromeVersion = parseInt(chromeManifest.minimum_chrome_version, 10);
+const minFirefoxVersion = parseInt(
+  firefoxManifest.applications.gecko.strict_min_version,
+  10,
+);
+validateVersion(minChromeVersion);
+validateVersion(minFirefoxVersion);
+
+function validateVersion(version) {
+  if (version > 0 && version < 200) {
+    return;
+  }
+  throw new Error('Suspicious browser version in manifest: ' + version);
+}
+
+module.exports = api => {
+  const isTest = api.env('test');
+  const targets = {};
+  if (isTest) {
+    targets.node = 'current';
+  } else {
+    targets.chrome = minChromeVersion.toString();
+    targets.firefox = minFirefoxVersion.toString();
+    // We won't support IE because that'll double profile import times.
```

Comment:
It's unfortunate that we have to "support" IE for the case of RN, particularly if this results in less optimal compiled code. Maybe we can address this in a more wholistic way.

1. I'll follow up with the Hermes team to see if this is still the best way to target Hermes for the standalone DevTools' embedded backend.
2. I'll post a PR (shortly) that kills this target for everything *except* that standalone backend. (That seems better than leaving it on for everything except the scheduling profiler.)

Reply:
Makes sense, thanks!